hash/casio_rompack.xml

```diff
@@ -240,12 +240,253 @@ license:CC0-1.0
+		<part name="cart" interface="casio_rompack">
+			<dataarea name="rom" size="0x1800">
+				<rom name="ro265.bin" size="0x1800" crc="a508af10" sha1="86422cab2e55633ab04dc12adc9ca3d4d6ade8c2"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!--
+	  6 songs
+	    1 Jingle Bells
+	    2 Silent Night
+	    3 Joy To The World
+	    4 Here Comes Santa Claus
+	    5 Silver Bells
+	    6 Santa Claus Is Comin' To Town
+	-->
+	<software name="ro270" supported="no">
+		<description>Christmas (RO-270)</description>
+		<year>1986</year>
+		<publisher>Casio / Sight &amp; Sound International Inc.</publisher>
+		<part name="cart" interface="casio_rompack">
+			<dataarea name="rom" size="0x1800">
+				<rom name="ro270.bin" size="0x1800" crc="654e8ba2" sha1="ffb682d1fe313567fbabad8b6e4e0497dca2dd14"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!--
+	  5 songs
+	    1 Bad Moon Rising
+	    2 Chantilly Lace
+	    3 The Dock Of The Bay
+	    4 I'm Walkin'
+	    5 Respect
+	-->
+	<software name="ro272" supported="no">
+		<description>Rock &amp; Roll Classics (RO-272)</description>
+		<year>1986</year>
+		<publisher>Casio / Sight &amp; Sound International Inc.</publisher>
+		<part name="cart" interface="casio_rompack">
+			<dataarea name="rom" size="0x1800">
+				<rom name="ro272.bin" size="0x1800" crc="1ddca979" sha1="7264cdeeea56194d4b8c37cffb5a9a45ace7c006"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!--
+	  4 songs
+	    1 Dancing In The Dark
+	    2 King For A Day
+	    3 These Dreams
+	    4 Walk Of Life
+	-->
+	<software name="ro274" supported="no">
+		<description>Hot Pop &amp; Rock (RO-274)</description>
+		<year>1987</year>
+		<publisher>Casio / Sight &amp; Sound International Inc.</publisher>
+		<part name="cart" interface="casio_rompack">
+			<dataarea name="rom" size="0x1800">
+				<rom name="ro274.bin" size="0x1800" crc="b174be9a" sha1="a34f8c72616d6dbe82df56ec1dab174606eafa15"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!--
+	  4 songs
+	    1 Theme From Hill Street Blues
+	    2 Theme From Cheers
+	    3 Theme From Dinasty
+	    4 Theme From St.Elsewhere
+	-->
+	<software name="ro276" supported="no">
+		<description>T.V. Tunes (RO-276)</description>
+		<year>1987</year>
+		<publisher>Casio / Sight &amp; Sound International Inc.</publisher>
+		<part name="cart" interface="casio_rompack">
+			<dataarea name="rom" size="0x1800">
+				<rom name="ro276.bin" size="0x1800" crc="be4240b7" sha1="08761ccbd6a467af1b61c3e7eea0c50028682344"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!--
+	  4 songs
+	    1 On The Sunny Side Of The Street
+	    2 Georgia On My Mind
+	    3 I Left My Heart In San Francisco
+	    4 Memories Of You
+	  The earlier (1984) edition of this pack is labelled "Jazz Standards" while the later (1986) edition is "Great Standards."
+	  There are apparently no other differences between the editions. 
+	-->
+	<software name="ro355" supported="no">
+		<description>Great Standards / Jazz Standards (RO-355)</description>
```

Comment:
Please use `alt_title` for alternate titles.

Reply:
Done, thanks.